.projenrc.ts

```diff
@@ -1,7 +1,7 @@
 import { ReleasableCommits, awscdk, github, javascript, release } from 'projen';
 import { ArrowParens, NodePackageManager } from 'projen/lib/javascript';
 
-let cdkVersion = '2.120.0';
+let cdkVersion = '2.149.0';
```

Comment:
we should try to keep this as low as possible. is there a need for this being 2.149.0 vs 2.120.0?

Reply:
I wan to use `DailyAutomaticBackupStartTime` class implemented in [v2.148.0](https://github.com/aws/aws-cdk/releases/tag/v2.148.0)

https://github.com/aws/aws-cdk/pull/30343

It is also possible to address this by describing the same content as `DailyAutomaticBackupStartTime` class in this PR while keeping it at version v2.120.0. Do you have any thoughts on this?

Reply:
We only want to bump CDK versions if really necessary and in a coordinated way. So CDK version bumps would be its own PR; then constructs can start using it

Reply:
Sure. I've added `daily-automatic-backup-start-time.ts`.